src/ort_genai_c.cpp

```diff
@@ -208,6 +208,34 @@ OgaResult* OGA_API_CALL OgaGenerator_GenerateNextToken(OgaGenerator* generator)
   OGA_CATCH
 }
 
+OgaResult* OGA_API_CALL OgaGenerator_GetOutput(const OgaGenerator* oga_generator, const char* name, OgaTensor** out) {
```

Comment:
OgaTensor doesn't have a destroy function. We need a function for it.

Reply:
I think we already have a destroy function for `OgaTensor`:

https://github.com/microsoft/onnxruntime-genai/blob/f3a77dadf94ea35e1e7b61f0533c1470137671f5/src/ort_genai_c.h#L286

Reply:
yes, it does. thanks